src/ballet/base64/fuzz_base64.c

```diff
@@ -0,0 +1,49 @@
+#if !FD_HAS_HOSTED
+#error "This target requires FD_HAS_HOSTED"
+#endif
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "../../util/fd_util.h"
+#include "fd_base64.h"
+
+int
+LLVMFuzzerInitialize( int  *   argc,
+                      char *** argv ) {
+  /* Set up shell without signal handlers */
+  putenv( "FD_LOG_BACKTRACE=0" );
+  fd_boot( argc, argv );
+  atexit( fd_halt );
+  return 0;
+}
+
+#define get_encoded_len(bytes) ( ( ( ( bytes*4UL + 2UL) / 3UL ) + 3UL ) & ~3UL )
+#define MAX_DATA_SZ 4096UL
+#define MAX_ENCODED_SZ ( get_encoded_len( MAX_DATA_SZ )+1UL )
+
+int
+LLVMFuzzerTestOneInput( uchar const * data,
```

Comment:
I feel like this fuzz target should be called fuzz_base64_roundtrip and that there should be another one attempting to decode trash.